homeassistant/components/enphase_envoy/config_flow.py

```diff
@@ -42,12 +42,12 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    def __init__(self):
+    def __init__(self) -> None:
         """Initialize an envoy flow."""
-        self.ip_address = None
+        self.ip_address: str | None = None
         self.username = None
```

Comment:
`self.username` doesn't seem to be assigned anywhere else AFAIC. It always stays `None`.

Reply:
I think we can even move these constants out of the constructor

Reply:
> I think we can even move these constants out of the constructor

Will leave that for another time then.